plugins/example-code-snippets/src/api-support/api-support-matrix-generator.ts

```diff
@@ -212,7 +212,7 @@ export class ApiSupportMatrixGenerator {
     for (const sdk of SDKS) {
       const sdkRepoDir = sourceProvider.sdkSourceDir(sdk.sdk);
       const sdkName = sdk.sdk.valueOf();
-      console.log(`Checking sdk: ${sdkName}`);
+      // console.log(`Checking sdk: ${sdkName}`);
```

Comment:
Mean to leave this commented out?

Reply:
yeah, as people start trying to use this stuff I think it may be useful for debugging.  for now.